projects/com.oracle.truffle.llvm.test/tests/c/alias/globalVar4.c

```diff
@@ -0,0 +1,17 @@
+#include <stdlib.h>
+
+int original = 1324342;
+extern int alias __attribute__((alias("original")));
+extern int alias2 __attribute__((alias("alias")));
+extern int alias3 __attribute__((alias("alias2")));
```

Comment:
Yes, I did not implement that since I did not need it for my use case. I added support for it in #165.